apps/etterlatte-api/src/main/kotlin/no/nav/etterlatte/samordning/vedtak/FoedselsnummerDto.kt

```diff
@@ -0,0 +1,5 @@
+package no.nav.etterlatte.samordning.vedtak
+
+data class FoedselsnummerDto(
+    val value: String,
+)
```

Comment:
Ville heller brukt `FoedselsnummerDTO` som vi har brukt i de andre endepunktene til `etterlatte-api`. Føles også mer naturlig som ekstern part å skrive `{ "foedselsnummer":"0123456789" }` i stedet for `{ "value":"0123456789" }` som er litt intetsigende

Reply:
true that, endrer